.eslintrc.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
 		es6: true,
 		node: true,
 	},
+	parser: "babel-eslint",
```

Comment:
Should we use https://github.com/babel/babel/tree/master/eslint/babel-eslint-parser instead? babel-eslint is now deprecated.

Reply:
Where do you see notice about babel-eslint being deprecated? I am not seeing it as such with "check-is-deprecated" or on searches. But if it does the same job, I guess it probably is better to use something under the `@babel` umbrella.

Reply:
See comment below.